Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# gem "rails"
+gem 'rspec'
```

Comment:
@henriqueh146 configura seu editor para adicionar EOF (End of file) talvez isso ocasione algum problema